app/src/main/res/values-de/strings.xml

```diff
@@ -19,26 +19,26 @@
     <string name="dialog_title_logout">Abmelden?</string>
     <string name="dialog_message_logout">Bist du sicher dass du dich abmelden möchtest?</string>
     <string name="dialog_title_reviews">Lade Bewertungen</string>
-    <string name="dialog_message_reviews">Bitte warten! Empfange Vorschauen.</string>
+    <string name="dialog_message_reviews">Bitte warten. Empfange Bewertungen.</string>
```

Comment:
Why would it be wrong? "Bitte warten." is a proper german sentence. You could also write it as "Bitte warten, empfange Bewertungen." but I don't thinks this looks better necessarily.